exonum-java-binding/common/src/main/proto/runtime.proto

```diff
@@ -65,10 +65,28 @@ message ExecutionError {
 message ExecutionStatus {
   oneof result {
     google.protobuf.Empty ok = 1;
-    ExecutionError error = 2;
+    ExecutionError2 error = 2;
   }
 }
 
+// fixme: A temporary replacement until the message is fixed in the core:
```

Comment:
Probably it would be better to use `TODO`(+ ticket link, if known) instead of `fixme` because IDEA has special indexing of that.

Reply:
Replaced with protos from the core.